### DISCUSSION TITLE
Can I configure the settings of window.showOpenFilePicker when useFsAccessApi is true.

### DISCUSSION THREAD

**Discussion by chairedevil** [react-dropzone/react-dropzone#1218] [Q&A] [answered]
https://developer.mozilla.org/en-US/docs/Web/API/Window/showOpenFilePicker
I want to config the description of allowed file types. like below.
Can I do this?
```
const pickerOpts = {
  types: [
    {
      description: 'test name',
      accept: {
        'application/zip': ['.zip']
      }
    },
  ],
  excludeAcceptAllOption: false,
  multiple: false
};
```

and, Can I set allowed file types with a value that does not exist in the MIME type?
```
useDropzone({
  accept: {
    'application/zip': ['.zip', '.zip.*'],
  }
})
```

Thank you for your support.

**Comment by rolandjitsu** [comment, marked as answer]
@chairedevil unfortunately no. That property cannot be configured.
